argoverse/utils/interpolate.py

```diff
@@ -184,8 +189,13 @@ def interp_arc(t: int, px: np.ndarray, py: np.ndarray) -> np.ndarray:
 
     # are px and py both vectors of the same length?
     assert px.size == py.size
+    if pz is not None:
+        assert px.size == pz.size
 
-    pxy = np.array((px, py)).T  # 2d polyline
+    if dim == 2:
+        pxy = np.array((px, py)).T  # 2d polyline
+    else:
+        pxy = np.array((px, py, pz)).T  # 3d polyline
```

Comment:
Naming of `pxy` is a bit misleading in the 3D case.

Reply:
agreed -- maybe just `pts` is a better name? or `wpts` for waypoints?